### PR TITLE
Feature/add country analysis screen

### PR DESCRIPTION
**Pull request by m1awicki**
Introduces new screen that presents various chart for each country.

### CODE REVIEW

File: app/src/main/java/com/mobilecoronatracker/ui/countrieslist/CountriesListViewModel.kt [hunk 1]
```diff
@@ -78,7 +79,7 @@ class CountriesListViewModel(
     }
 
     override fun onCountryAnalysisRequested(country: String) {
```
Comment (MDikkii) [inline app/src/main/java/com/mobilecoronatracker/ui/countrieslist/CountriesListViewModel.kt]: Maybe `countryName`?

File: app/src/main/java/com/mobilecoronatracker/ui/countrieslist/FragmentCountriesList.kt [hunk 2]
```diff
@@ -95,5 +96,11 @@ class FragmentCountriesList : Fragment() {
             adapter.countriesReports = it
             adapter.notifyDataSetChanged()
         })
+        viewModel.navigationToCountryRequested.observe(viewLifecycleOwner, Observer {
+            if (it.isBlank()) return@Observer
```
Comment (MDikkii) [inline app/src/main/java/com/mobilecoronatracker/ui/countrieslist/FragmentCountriesList.kt]: I would prefer brackets when it is not if else inline statement

File: app/src/main/java/com/mobilecoronatracker/ui/countryanalysis/CountryAnalysisViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+    private suspend fun loadCountryInfo(countryName: String) {
+        val country = countriesDao.getByCountryName(countryName)
+        country?.let {
+            countryFlagPath.postValue(it.countryFlagPath)
+            this.countryName.postValue(it.name)
+        }
+    }
+
+    private fun onCountryHistory(history: List<CountryReportTimePointModelable>) {
```
Comment (MDikkii) [inline app/src/main/java/com/mobilecoronatracker/ui/countryanalysis/CountryAnalysisViewModel.kt]: Very long method ;)

File: gradle.properties [hunk 4]
```diff
@@ -19,3 +19,5 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+android.useAndroidX=true
```
Comment (MDikkii) [inline gradle.properties]: It is already defined in this file

**Review by MDikkii — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 4 above)
Comment (m1awicki) [reply gradle.properties]: lol, I must be blind xD

File: app/src/main/java/com/mobilecoronatracker/ui/countrieslist/FragmentCountriesList.kt (hunk 2 above)
Comment (m1awicki) [reply app/src/main/java/com/mobilecoronatracker/ui/countrieslist/FragmentCountriesList.kt]: done

File: app/src/main/java/com/mobilecoronatracker/ui/countrieslist/CountriesListViewModel.kt (hunk 1 above)
Comment (m1awicki) [reply app/src/main/java/com/mobilecoronatracker/ui/countrieslist/CountriesListViewModel.kt]: ok

File: app/src/main/java/com/mobilecoronatracker/ui/countryanalysis/CountryAnalysisViewModel.kt (hunk 3 above)
Comment (m1awicki) [reply app/src/main/java/com/mobilecoronatracker/ui/countryanalysis/CountryAnalysisViewModel.kt]: fixed

**Review by MDikkii — APPROVED** [review]

**Pull request merged**